Add email postponing competition (#48)
srobo/team-emails: SR2020/2020-03-16-competition-postpone.md

---
to: sr2020-teams
subject: Postponing Student Robotics 2020
---

Hello all,

> Following guidance from the UK government and the University of Reading, we will need to postpone Student Robotics 2020 from the original date in April, until some time late June, early July.

Reviewer: ```suggestion
Following guidance from the UK government and the University of Reading, we will need to postpone Student Robotics 2020 from the original date in April, until some time late June/early July.
```

## Competition

> 

Reviewer 2: [NIT] Worth listing the dates so they don't need to enter the form to see them?

Please fill in the below doodle poll to let us know your preferred dates.

Doodle Poll: https://doodle.com/poll/rmdg8waeezbdqbz9

The event will still take place over a weekend, at the University of Reading.

> We hope to confirm the updated date in the next few weeks, so please let us know as soon as you can.

Reviewer 2: ```suggestion
We hope to confirm the updated date in the next week, so please let us know as soon as you can.
```
Add pressure, also the trustees suggested this decision be made quickly

## Tech Days

There are no current planned tech days. Once things have settled down, we can organise some.

The [forum][forum] is still a great place to get help from blueshirts and other teams, in addition to showing off your progress!

If you are looking for more in-depth support from blueshirts, you can [email us][teams-email] to request remote mentoring.

> If you have any further questions, please [let us know][teams-email].

Reviewer 2: [NIT] we should probably make the address visible. just so people can see it without launching their email app, 

```suggestion
If you have any further questions, please [let us know by emailing teams@studentrobotics.org][teams-email].
```
(Same with usage above)

---
to: sr2020-teams

> subject: Postponing Student Robotics 2020

Reviewer 3: perhaps be clear that this is the competition event that's being postponed? Not sure if that matters here, since the competition year has already started.

Author: I too was wondering this, yea can't hurt!

---

Hello all,

Following guidance from the UK government and the University of Reading, we will need to postpone Student Robotics 2020 from the original date in April, until some time late June / early July.

## Competition

We are currently investigating the following dates for the competition:

- 27th - 28th June
- 4th - 5th July
- 11th - 12th July

> Please fill in the following poll to let us know your preferred dates: https://forms.gle/21x6F5EBMAtu6GKh9

Reviewer 3: Maybe duplicate the introduction text onto the form too, just so it's clear?

Author: Done

The event will still take place over a weekend, at the University of Reading.

We hope to confirm the updated date in the next week, so please let us know as soon as you can.

## Tech Days

> There are no current planned tech days. Once things have settled down, we can organise some.

Reviewer 3: Is this realistic given the current government advice?

If so, perhaps use "may" or "hope to" rather than "can".

The [forum][forum] is still a great place to get help from blueshirts and other teams, in addition to showing off your progress!

If you are looking for more in-depth support from blueshirts, you can email us at teams@studentrobotics.org to request remote mentoring.

> 

Reviewer 3: redundant double newline here, feel free to ignore though.
